app/controllers/spree/admin/orders_controller.rb

```diff
@@ -101,6 +102,13 @@ def print_ticket
 
       private
 
+      def first_completed_payment
+        if @order.payments.any?
+          @order.payments.select { |p| p.state == "checkout" || p.state == "completed" }
```

Comment:
are we sure this is not already implemented somewhere? I also wonder why we need `checkout`. That's not a complete payment. There may be something else these method names and logic are not telling me :thinking:

Reply:
> are we sure this is not already implemented somewhere?

I had a look, and cannot find anything anywhere... But maybe I'm not looking for the right thing.

> I also wonder why we need checkout. That's not a complete payment.

When i tested an order process as a consumer my order was with a payment with a `checkout` state. That's also what Kirsten mentioned here: https://github.com/openfoodfoundation/openfoodnetwork/issues/6359#issuecomment-832992857

Reply:
> we want to show is not the first completed payment but its information, regardless of its state.

If I understand correctly, the state is important to display this information.
Eg:
Having an order, during the checkout:
 - I made a payment with my credit card, which failed
 - I made a payment 'Cash on collection' that, obviously, is successful

As the result, the information that must be displayed is the one linked to the second payment, not the first one (failed).

Am I wrong? Sorry, it's relatively new to me, I may miss something around the understanding of the checkout, orders, payments, ...

Reply:
That's where I'm afraid we would end up: it's not clear what we need to build and we should clarify with @openfoodfoundation/train-drivers-product-owners . We need to decide whether we show completed payments (there might be many) or show all of them regardless of their state. I personally lean towards the first one.

Reply:
Can we chose a solution that is consistent elsewhere in the platform? Today the payment info shows up in the order confirmation and the order confirmation email. Can't we just reuse that same info on the invoice?

Reply:
That sounds like a sensible choice :+1: thoughts @jibees ?

Reply:
Code that is already used: 

https://github.com/jibees/openfoodnetwork/blob/6359-add-payment-method-description-to-invoice/app/helpers/order_helper.rb#L5

Reply:
Code that is already used: https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/helpers/order_helper.rb#L5

Will use the same ;)

Reply:
Way way simpler now.